scripts/generate_env_vars.sh

```diff
@@ -33,6 +33,50 @@ function gen_random_string {
     openssl rand -hex 16 | tr -d "\n"
 }
 
+function ccloud_admin {
+    if [ "$AETHER_CONNECT_MODE" = 'CONFLUENT' ]; then
+        cat << EOF_CC
+# ------------------------------------------------------------------
+
+
+# ------------------------------------------------------------------
+# Confluent Cloud Admin
+# ==================================================================
+CC_API_USER=${CC_API_USER}
+CC_API_PASSWORD=${CC_API_PASSWORD}
+CC_CLUSTER_NAME=${CC_CLUSTER_NAME}
+
+
+EOF_CC
+    else
+        echo ""
+    fi
+}
+
+function kafka_settings {
+    if [ "$AETHER_CONNECT_MODE" = 'CONFLUENT' ]; then
+        cat << EOF1
+KAFKA_URL=${CC_URL}
+KAFKA_SECURITY=SASL_SSL
+# kafka all-tenant Superuser
+KAFKA_SU_USER=${CC_SU_USER}
+KAFKA_SU_PASSWORD=${CC_SU_PASSWORD}
+# default number of replicas to maintain
+KAFKA_REPLICAS=3
+EOF1
+    else
+        cat << EOF2
+KAFKA_URL=kafka:29092
+KAFKA_SECURITY=SASL_PLAINTEXT
+# kafka all-tenant Superuser
+KAFKA_SU_USER=master
+KAFKA_SU_PASSWORD=${SERVICES_DEFAULT_ADMIN_PASSWORD:-adminadmin}
+# default number of replicas to maintain
+KAFKA_REPLICAS=1
+EOF2
+fi
```

Comment:
In the `.env` file a would include both setting together something like 

```properties
# ------------------------------------------------------------------
# Kafka & Zookeeper
# ==================================================================

# Kafka generic settings
KAFKA_URL=${CC_URL}
KAFKA_SU_USER=${CC_SU_USER}
KAFKA_SU_PASSWORD=${CC_SU_PASSWORD}

# ------------------------------------------------------------------
# Kakfa LOCAL mode (comment it out in CONFLUENT mode)
# ==================================================================
KAFKA_SECURITY=SASL_PLAINTEXT

# Internal root user
KAFKA_ROOT_USER=root
KAFKA_ROOT_PASSWORD=$(gen_random_string)

# default number of replicas to maintain
KAFKA_REPLICAS=1

# secret to generate tenant specific passwords
KAFKA_SECRET=$(gen_random_string)

# ZK settings
ZOOKEEPER_ROOT_USER=zk-admin
ZOOKEEPER_ROOT_PASSWORD=$(gen_random_string)

# ------------------------------------------------------------------
# Kafka CONFLUENT mode (comment it out in LOCAL mode)
# ==================================================================
# KAFKA_SECURITY=SASL_SSL
# # default number of replicas to maintain
# KAFKA_REPLICAS=3

# Confluent Cloud Admin
CC_API_USER=${CC_API_USER}
CC_API_PASSWORD=${CC_API_PASSWORD}
CC_CLUSTER_NAME=${CC_CLUSTER_NAME}
```

Reply:
I've gone a ways in this direction, but it still seems to need two blocks.